serialize/Cargo.toml

```
@@ -38,5 +41,7 @@ ark-test-curves = { workspace = true, default-features = false, features = [
 [features]
 default = []
 parallel = ["rayon"]
-std = ["ark-std/std"]
+std = ["ark-std/std", "serde_with/std"]
```

I'd prefer it not be a default feature, as then every dependent crate would immediately depend on `serde`. 
```suggestion
std = ["ark-std/std"]
```

---

Oh nvm, I see what's going on here. You might need to use the optional feature syntax here.
```suggestion
std = ["ark-std/std", "serde_with?/std"]
```

---

See here: https://doc.rust-lang.org/cargo/reference/features.html#dependency-features